pkg/apis/cluster/v1alpha1/cluster.go

```diff
@@ -61,20 +61,23 @@ type CNIConfig struct {
 type CustomClusterPhase string
 
 const (
-	// RunningPhase represents the cluster on VMs is creating. In this phase, the worker named ends in "init" is running to initialize the cluster on VMs
-	RunningPhase CustomClusterPhase = "Running"
+	// PendingPhase represents the customCluster's first phase after being created
+	PendingPhase CustomClusterPhase = "Pending"
 
-	// SucceededPhase represents the cluster on VMs is created successful. In this phase, the worker named ends in "init" is completed
-	SucceededPhase CustomClusterPhase = "Succeeded"
+	// ProvisioningPhase represents the cluster on VMs is provisioning. In this phase, the worker named ends in "init" is running to initialize the cluster on VMs
```

Comment:
nit: `worker named ends in "init"` not clear to me.

Reply:
If you mean the pod, can remove the saying here, because we do not need to expose implement details in API Desc